src/components/core/pivotTable/ResizedColumnsMap.ts

```diff
@@ -0,0 +1,183 @@
+// (C) 2007-2020 GoodData Corporation
+// tslint:disable:prefer-conditional-expression
+import { Execution } from "@gooddata/typings";
+import { ColDef, Column } from "ag-grid-community";
+import omit = require("lodash/omit");
+
+import { getColumnIdentifier, isMeasureColumn } from "./agGridUtils";
+import { getColumnWidthsFromMap, convertColumnWidthsToMap } from "./agGridColumnSizing";
+import {
+    ColumnEventSourceType,
+    ColumnWidthItem,
+    isAllMeasureColumnWidthItem,
+    IAllMeasureColumnWidthItem,
+    ColumnWidth,
+    IResizedColumnsItem,
+    isColumnWidthValue,
+    isColumnWidthAuto,
+} from "../../../interfaces/PivotTable";
+
+export interface IResizedColumnsCollection {
+    [columnIdentifier: string]: IResizedColumnsCollectionItem;
+}
+
+export interface IResizedColumnsCollectionItem {
+    width: ColumnWidth;
+    source: ColumnEventSourceType;
+}
+
+export class ResizedColumnsMap {
+    private manuallyResizedColumns: IResizedColumnsCollection;
+    private allMeasureColumnWidth: number | null;
+
+    public constructor() {
+        this.manuallyResizedColumns = {};
+        this.allMeasureColumnWidth = null;
+    }
+
+    public getManuallyResizedColumn(item: Column | ColDef): IResizedColumnsItem {
```

Comment:
I would prefer to have type signature with `?`, as it can return void.

Reply:
done